homeassistant/components/velux/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""
+
+Connects to VELUX KLF 200 interface
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.velux/
+"
+"""
+
+import logging
+import asyncio
+from homeassistant.helpers import discovery
+
+DOMAIN = "velux"
+SUPPORTED_DOMAINS = ['scene']
+_LOGGER = logging.getLogger(__name__)
+VELUX_MODULE = None
+
+REQUIREMENTS = ['pyvlx==0.1.1']
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Setup device tracker."""
+
+    # pylint: disable=global-statement, import-error
+    global VELUX_MODULE
+
+    if VELUX_MODULE is None:
+        VELUX_MODULE = VeluxModule(hass, config)
+        yield from VELUX_MODULE.async_start()
```

Comment:
I guess that there will be an exception if the host is not reachable and/or the password is wrong.